crm_sector/models/crm_lead.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingenieria S.L. - Javier Iniesta
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api, exceptions, _
+
+
+class CrmLead(models.Model):
+    _inherit = 'crm.lead'
+
+    sector_id = fields.Many2one(comodel_name='res.partner.sector',
```

Comment:
If you are migrating from https://github.com/Antiun/antiun-odoo-addons/tree/8.0/crm_sector, you should add `oldname="sector"` to make happy your sysadmin.